presto-orc/src/main/java/com/facebook/presto/orc/OrcOutputBuffer.java

```diff
@@ -91,7 +91,6 @@ public OrcOutputBuffer(ColumnWriterOptions columnWriterOptions, Optional<DwrfDat
         this.buffer = new byte[INITIAL_BUFFER_SIZE];
         this.slice = wrappedBuffer(buffer);
 
-        compressedOutputStream = new ChunkedSliceOutput(MINIMUM_OUTPUT_BUFFER_CHUNK_SIZE, MAXIMUM_OUTPUT_BUFFER_CHUNK_SIZE);
```

Comment:
Initialization of the output buffer moved from the constructor to the `writeChunkToOutputStream` where the write is happening.